x-pack/test/cases_api_integration/common/lib/api/user_actions.ts

```diff
@@ -7,51 +7,29 @@
 
 import {
   getCaseFindUserActionsUrl,
-  getCaseUserActionUrl,
```

Comment:
Should we also delete the declaration of `getCaseUserActionUrl` in`x-pack/platform/plugins/shared/cases/common/api/helpers.ts`? I think it is not used anywhere anymore

Reply:
done